### ISSUE TITLE
workflow icon: type check failed for prop "status"

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#1169]
Seemingly harmless traceback which shows up in the console from time to time.

This means one of the workflow icon components is getting `undefined` passed in as the `status` which either means:
* The status isn't being requested.
* The icon is being rendered before the data is ready.

```
[Vue warn]: Invalid prop: type check failed for prop "status". Expected String, got Undefined 

found in

---> <WorkflowIcon> at src/components/cylc/gscan/WorkflowIcon.vue
       <TreeItem> at src/components/cylc/tree/TreeItem.vue... (1 recursive calls)
         <Tree> at src/components/cylc/tree/Tree.vue
           <VSkeletonLoader>
             <GScan> at src/components/cylc/gscan/GScan.vue
               <VList>
                 <VNavigationDrawer>
                   <Drawer> at src/components/cylc/Drawer.vue
                     <Default> at src/layouts/Default.vue
                       <VApp>
                         <App> at src/App.vue
                           <Root>
```

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by MetRonnie** [comment]
Is this still applicable? I haven't seen this; propose closing it

**Comment by oliver-sanders** [comment]
I suspect that the issue is still there but presenting itself differently (I think we have a fallback icon now?).

Can probably close as superseded by:

https://github.com/cylc/cylc-uiserver/issues/485

**Comment by MetRonnie** [comment]
Can re-open if seen again
